specification/flag-evaluation/flag-evaluation.md

```diff
@@ -0,0 +1,195 @@
+# Flag Evaluation API
+
+## Overview
+
+The `evaluation API` allows for the retrieval of feature flag values, independent of any flag control plane or vendor. In the absence of a feature flag system implementation (a [provider](../provider/providers.md)) the `evaluation API` uses the "No-op provider", which simply returns the supplied default flag value.
```

Comment:
I would either use the term "feature flag system implementation" or "provider" and not both. If you insist on using both I would put it the other way around as "provider" is the official name and "feature flag system implementation" is a description of what it is. IMO the link to the provider spec explains it sufficiently though.